iphone/TitaniumKit/TitaniumKit/Sources/Kroll/KrollContext.m

```diff
@@ -793,6 +793,10 @@ - (void)main
 {
   pthread_mutex_lock(&KrollEntryLock);
   context = JSGlobalContextCreate(NULL);
+  // Ensure that the JSContext is debuggable
+#if __IPHONE_OS_VERSION_MAX_ALLOWED >= 160400
+  JSGlobalContextSetInspectable(context, true);
```

Comment:
Small thing: In the ObjC world, `YES` is favored over `true` and `TRUE`.

Besides that, great change! It's just a bit complex to disable this per target, as TitaniumKit is precompiled during the SDK build, but I think we have some prod-related checks that we could reuse here. Checking …

Reply:
Might found a solution:
```objc
  BOOL isProduction = [TiApp.tiAppProperties[@"ti.deploytype"] isEqualToString:@"production"];
```

Reply:
Just pushed c70ebd011232460f4ad4dd2b1e3fab6b310995d9 with those changes. I don't have production builds setup, but I was able to test that changing to `if (isProduction) {`  causes the JSContext to not be inspectable in dev builds so I think it should be good

Reply:
@ewanharris That one didn't work, yet, as it's deprecated. So if users don't manually add `<string name="ti.deploytype"">production</string>` in their tiapp.xml, it will not work. Really hard to get these config so early in the app lifecycle

Reply:
Ah shoot :/ I think we can defer the setting of this potentially to be later in the lifecycle if needed as it can be toggled on/off AFAIK. Is there a place that might be better to do that?

Feel free to push to the PR if you wanna rework it at all

Reply:
I tried some places already, which all worked in dev, but crashed in prod builds (tested via testflight). I'll revisit it tomorrow 👍

Reply:
Found something promising! Doing some production tests and then push here.